HealthcareSystem/PatientWebApp/wwwroot/js/activate_account.js

```diff
@@ -0,0 +1,26 @@
+﻿$(document).ready(function () {
+    $("#activate").click(function (e) {
+        e.preventDefault();
+
+        let url_split = window.location.href.split("?")[1];
+
+        var jmbg = url_split.split("=")[1];
+
+        $.ajax({
```

Comment:
Asking just because you said you didn't want to have the page with the activation button but couldn't get it to work otherwise, what exactly happens if you just put the jmbg extraction and ajax directly into $(document).ready instead of $("#activate").click?

Reply:
This worked, applied in ba19e00e8e1fc56824a0e731412f2cafdc3c5403.